src/fragments/ChatMessages.tsx

```diff
@@ -109,15 +109,14 @@ const ChatMessages: React.FC<ChatMessagesProps> = ({
             key={message.id}
             avatar={
               isUserMessage(message) ? (
-                <Avatar
-                  variant={AvatarVariants.ROUND}
-                  bg={getUniqueColor(
-                    message.user.displayName,
-                    theme.colors.avatars
-                  )}
-                >
-                  {message.user.displayName[0]}
-                </Avatar>
+                <img 
```

Comment:
You should use `UserAvatar` here as well

Reply:
Should be changed now